### DISCUSSION TITLE
current external generator power

### DISCUSSION THREAD

**Discussion by diomedon** [weltenwort/home-assistant-rct-power-integration#307] [Q&A]
Hi,
I have a second inverter as a slave installed. I get the daily, weekly, etc. via RCT Power Storage External Energy Production Day / ...

However there is no option get current power generated by the 2nd inverter, at least I do not see one. Any idea?

via a similar IO broker adapter, this information is available.
Regards

Carsten

**Comment by weltenwort** [comment]
Hi @diomedon,

is the io broker adapter's source code available so we can take a look which register they're using?

**Reply by diomedon** [reply]
Hi @weltenwort 

https://github.com/aruttkamp/ioBroker.rct

This one. Does this help?

Regards

Carsten

**Reply by diomedon** [reply]
@weltenwort Hi Felix, is there a chance that current external power will be supported anytime soon?
Regards
Carsten

**Reply by weltenwort** [reply]
@diomedon I'd love to add support, but I'm just not able to find the spare time right now. So it's on my to-do list, but I can't give any promises regarding the timing, sorry.

The iobroker link helps somewhat, thanks. But I still couldn't identify the exact register that contains that information. Can you tell me what the name of the iobroker state is?

On a separate thought: Is your second inverter also from RCT? If so, you could install a second instance of this integration that connects to it too and collects the value directly from it.

**Reply by diomedon** [reply]
Hi, thanks. I thought about connecting the second inverter directly. It might work, both work in some sort of master and slave mode, but I think the 2nd inverter pushes its data to the main one, so the modbus port of the 2nd inverter is not occupied by internal communication. 
Regards
Carsten
